client/src/pages/Step/index.js

```diff
@@ -21,20 +24,53 @@ import { GENERAL } from '../../constants/nav-routes';
 
 const { Checkbox } = Inputs;
 const { Row, Col } = Grid;
+const { Tips, Checklist } = Cards;
+
+const Step = () => {
+  const [checklist, setChecklist] = useState({
+    thingsYouWillNeed: [],
+    whatYouWillNeedToKnow: [],
+  });
```

Comment:
can't this be added to `useSteps` custom hook? and make the useSteps return them?

Reply:
done as part of connecting